jib-core/src/main/java/com/google/cloud/tools/jib/http/Connection.java

```diff
@@ -23,23 +23,25 @@
 import com.google.api.client.http.HttpResponse;
 import com.google.api.client.http.apache.ApacheHttpTransport;
 import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.common.base.Preconditions;
 import java.io.Closeable;
 import java.io.IOException;
 import java.net.URL;
 import javax.annotation.Nullable;
 import org.apache.http.NoHttpResponseException;
 
 /**
- * Sends an HTTP {@link Request} and stores the {@link Response}.
+ * Sends an HTTP {@link Request} and stores the {@link Response}. Clients should not send more than
+ * one request.
  *
  * <p>Example usage:
  *
- * <pre>{@code
+ * <pre>
```

Comment:
I believe the `{@code` is the recommended way of wrapping multi-line code blocks. (https://stackoverflow.com/a/542142/10047775)

Reply:
TIL